README.md

```diff
@@ -52,7 +52,7 @@ The latest version of neofs-node works with neofs-contract
 
 # Building
 
-To make all binaries you need Go 1.17+ and `make`:
+To make all binaries you need Go 1.20+ and `make`:
```

Comment:
maybe say that you need at least version from go.mod? not sure we can keep it actual

Reply:
We keep it actual in NeoGo, it’s just you have to remember to update it while updating go.